### ISSUE TITLE
Yaks operator not found

### ISSUE THREAD

**Issue by nicolaferraro** [citrusframework/yaks#179]
Tested on Release 0.1.0-202011100006 (Release 0.1.0-202011110004 has no artifacts):

```
$ yaks test sources.feature 
Creating new test namespace yaks-081318a4-7c66-4375-8b6e-440b175cc1c9
AutoRemove namespace yaks-081318a4-7c66-4375-8b6e-440b175cc1c9
Test results: Total: 0, Passed: 0, Failed: 0, Skipped: 0

Error: deployments.apps "yaks-operator" not found
```

**Comment by christophd** [comment]
is this on a `global` yaks installation? how did you install yaks on that cluster?

**Comment by christophd** [comment]
please note: when installing yaks on OpenShift please use `--olm=false` option (even if OLM is available on your cluster) as the installer will try to use OLM installation by default. The OLM installation will fail silently though because the YAKS operator has not been added to OLM yet.

**Comment by nicolaferraro** [comment]
No it wasn't OpenShift, but I didn't install it in advance and that was the cause. I think the error can be more descriptive here ;)

**Comment by christophd** [comment]
let me see if I can improve the error message for that

**Comment by christophd** [comment]
should be fixed with #205
